### ISSUE TITLE
`set-env` command should not be used

### ISSUE THREAD

**Issue by pavelkryukov** [petarpetrovt/setup-sde#11]
Hi, I have a following error while using your action:

```
/usr/bin/xvfb-run --auto-servernum node /home/runner/work/_actions/petarpetrovt/setup-sde/v0.8/installer/index.js
Error: Unable to process command '::set-env name=SDE_PATH::/home/runner/work/putty-ci/putty-ci/.output/sde-temp-files/sde-external-8.59.0-2020-10-05-lin' successfully.
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

https://github.com/pavelkryukov/putty-ci/runs/1546864585?check_suite_focus=true#step:13:26

**Comment by petarpetrovt** [comment]
Thanks for reporting, should be fixed in `v0.9`

**Comment by pavelkryukov** [comment]
Thank you!
